README.md

```diff
@@ -41,9 +41,9 @@ That's not a question, it's an order! Or more of a friendly offer, really. Kanda
 
 * GitHub [issues tracker](https://github.com/kandanapp/kandan/issues)
 * Twitter [@kandanapp](https://twitter.com/kandanapp)
-* [Kandan Demo](http://kandan-demo.kandanapp.com) (Yes we eat our own DogFood)
+* [Kandan Demo](http://demo.getkandan.com) (Yes we eat our own DogFood)
 
-You can also email us at [admin@kandanapp.com](mailto:admin@kandanap.com)
```

Comment:
This email actually would have never worked! (admin@kandan*ap*.com vs admin@kandan*app*.com)